docs/mix_configs.cheatmd

```diff
@@ -0,0 +1,14 @@
+## Mix Configs
+
+Mix Config files have their config stanzas sorted. Similar to the sorting of aliases, this delivers consistency to an otherwise arbitrary world, and can even help catch bugs like configuring the same key multiple times.
+
+A file is considered a config file if
+
+1. its path matches `config/.*\.exs` or `rel/overlays/.*\.exs`
+2. the file imports Mix.Config (`import Mix.Config`)
```

Comment:
```suggestion
2. the file imports Config (`import Config` or the deprecated `Mix.Config`)
```

Reply:
ha actually it doesn't run on `import Mix.Config` xD